Code/client/Components/PlayerSummonComponent.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
```

Comment:
Why PlayerSummonComponent?

Reply:
I expected I would need to store more data to know who the owner is. Turned out to be kinda unnecessary. I intend to refactor this into something simpler

Reply:
Ah, I've been refactoring it a bit already, I'll push it in a bit.